query-engine/driver-adapters/js/smoke-test-js/src/client/client.ts

```diff
@@ -18,6 +18,7 @@ export async function smokeTestClient(connector: ErrorCapturingConnector) {
     describe(isUsingJsConnector ? `using JS Connectors` : `using Rust drivers`, () => {
       it('batch queries', async () => {
         const prisma = new PrismaClient({
+          // @ts-ignore
           jsConnector,
```

Comment:
Missing rename? for
- `isUsingJsConnector`
- `using JS Connectors`
- `jsConnector`

Reply:
Possibly even missing adaptation to new DX.

Reply:
Jan, if by "missing adaptation" you refer to `PrismaClient` accepting a `jsConnector` argument rather than `adapter`, that's because this PR will be used to update `@prisma/client`, and not viceversa. I've mentioned follow-up PRs in this PR description :)

Joel, thanks for spotting the missing renames.